cargo-pgx/src/commands/init.rs

```diff
@@ -275,7 +280,24 @@ fn make_install_postgres(version: &PgConfig, pgdir: &PathBuf) -> Result<PgConfig
     let output = child.wait_with_output()?;
 
     if output.status.success() {
-        let mut pg_config = get_pg_installdir(pgdir);
+        let mut pgx_versioned_dir = pgx_dir.clone();
+        let versioned_dir = format!(
+            "{}.{}",
+            version.major_version()?,
+            version.minor_version()?
+        );
+        pgx_versioned_dir.push(&versioned_dir);
+        // The longest part is over, move the folder into position.
+        println!(
+            "{} {} with {}",
+            "   Replacing".bold().green(),
```

Comment:
need extra leading space to line things up correctly.  obviously, this is the most important thing ever.